server/migrations/20230325030839-add-token-id-to-user-authentications.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn("user_authentications", "oidcTokenId", {
```

Comment:
It's a shame to have another column just for this, but hard to see a way to avoid it.